specification/liftrneon/Neon.Postgres.Management/main.tsp

```diff
@@ -74,124 +74,64 @@ interface Organizations {
-}
-
-@doc("The Project resource type.")
-@added(Neon.Postgres.Versions.v2_preview)
-@parentResource(OrganizationResource)
-model Project is ProxyResource<Models.ProjectProperties> {
-  ...ResourceNameParameter<Project>;
+    connectionUriParameters: ConnectionUriProperties,
+  ): ArmResponse<ConnectionUriProperties> | ErrorResponse;
 }
 
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface Branches {
   get is ArmResourceRead<Branch>;
   createOrUpdate is ArmResourceCreateOrReplaceAsync<Branch>;
-  update is ArmResourcePatchAsync<Branch, Models.BranchProperties>;
   delete is ArmResourceDeleteSync<Branch>;
   list is ArmResourceListByParent<Branch>;
 }
 
-@doc("The Branch resource type.")
-@added(Neon.Postgres.Versions.v2_preview)
-@parentResource(Project)
-model Branch is ProxyResource<Models.BranchProperties> {
-  ...ResourceNameParameter<Branch>;
-}
-
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface Computes {
-  get is ArmResourceRead<Compute>;
-  createOrUpdate is ArmResourceCreateOrReplaceAsync<Compute>;
-  update is ArmResourcePatchAsync<Compute, Models.ComputeProperties>;
-  delete is ArmResourceDeleteSync<Compute>;
   list is ArmResourceListByParent<Compute>;
 }
 
-@doc("The Compute resource type.")
-@added(Neon.Postgres.Versions.v2_preview)
-@parentResource(Branch)
-model Compute is ProxyResource<Models.ComputeProperties> {
-  ...ResourceNameParameter<Compute>;
-}
-
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface NeonDatabases {
-  get is ArmResourceRead<NeonDatabase>;
-  createOrUpdate is ArmResourceCreateOrReplaceAsync<NeonDatabase>;
-  update is ArmResourcePatchAsync<NeonDatabase, Models.NeonDatabaseProperties>;
-  delete is ArmResourceDeleteSync<NeonDatabase>;
   list is ArmResourceListByParent<NeonDatabase>;
 }
 
-@doc("The Neon Database resource type.")
-@added(Neon.Postgres.Versions.v2_preview)
-@parentResource(Branch)
-model NeonDatabase is ProxyResource<Models.NeonDatabaseProperties> {
-  ...ResourceNameParameter<NeonDatabase>;
-}
-
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface NeonRoles {
-  get is ArmResourceRead<NeonRole>;
-  createOrUpdate is ArmResourceCreateOrReplaceAsync<NeonRole>;
-  update is ArmResourcePatchAsync<NeonRole, Models.NeonRoleProperties>;
-  delete is ArmResourceDeleteSync<NeonRole>;
   list is ArmResourceListByParent<NeonRole>;
 }
 
-@doc("The Neon Role resource type.")
-@added(Neon.Postgres.Versions.v2_preview)
-@parentResource(Branch)
-model NeonRole is ProxyResource<Models.NeonRoleProperties> {
-  ...ResourceNameParameter<NeonRole>;
-}
-
 @armResourceOperations
 @added(Neon.Postgres.Versions.v2_preview)
 interface Endpoints {
-  get is ArmResourceRead<Endpoint>;
-  createOrUpdate is ArmResourceCreateOrReplaceAsync<Endpoint>;
-  update is ArmResourcePatchAsync<Endpoint, Models.EndpointProperties>;
-  delete is ArmResourceDeleteSync<Endpoint>;
```

Comment:
Why there are so many operations removed in a bugfix pr? Is this by intention or a mistake?